patterns/2-structured/repository-activity-score.md

```diff
@@ -58,17 +58,23 @@ function calculateScore(repo) {
     // initial score is 50 to give active repos with low GitHub KPIs (forks, watchers, stars) a better starting point
     let iScore = 50;
     // weighting: forks and watches count most, then stars, add some little score for open issues, too
-    iScore += repo["forks_count"] * 5 + repo["watchers_count"] + repo["stargazers_count"] / 3 + repo["open_issues_count"] / 5;
-    let iDaysSinceLastUpdate = (new Date().getTime() - new Date(repo.updated_at).getTime()) / 1000 / 86400;
+    iScore += repo.forks_count * 5;
+    iScore += (repo.subscribers_count ? repo.subscribers_count : 0);
+    iScore += repo.stargazers_count / 3;
+    iScore += repo.open_issues_count / 5;
+
     // updated in last 3 months: adds a bonus multiplier between 0..1 to overall score (1 = updated today, 0 = updated more than 100 days ago)
-    iScore = iScore * (1 + (100 - Math.min(iDaysSinceLastUpdate, 100)) / 100);
+    let iDaysSinceLastUpdate = (new Date().getTime() - new Date(repo.updated_at).getTime()) / 1000 / 86400;
+    iScore = iScore * ((100 - Math.min(iDaysSinceLastUpdate, 100)) / 100);
+
     // evaluate participation stats for the previous  3 months
     repo._InnerSourceMetadata = repo._InnerSourceMetadata || {};
     if (repo._InnerSourceMetadata.participation) {
         // average commits: adds a bonus multiplier between 0..1 to overall score (1 = >10 commits per week, 0 = less than 3 commits per week)
-        let iAverageCommitsPerWeek = repo._InnerSourceMetadata.participation.slice(repo._InnerSourceMetadata.participation - 13).reduce((a, b) => a + b) / 13;
-        iScore = iScore * (1 + (Math.min(Math.max(iAverageCommitsPerWeek - 3, 0), 7)) / 7);
+        let iAverageCommitsPerWeek = repo._InnerSourceMetadata.participation.slice(repo._InnerSourceMetadata.participation.length - 13).reduce((a, b) => a + b) / 13;
+        iScore = iScore * ((Math.min(Math.max(iAverageCommitsPerWeek - 3, 0), 7)) / 7);
```

Comment:
Hi , @3cpt! I agree the comment with 0..1 is a bit irritating, but on top of whats already there, we want to add a multiplier that scales the result based on the frequency of the current commits. If we remove the  "1+ multiplier" then the whole result is scaled down instead of adding something on top (e.g.  100 * 0.5 = 50 whereas 500 * (1 + 0.5) = 750). Therefore, let's keep the implementation as is for this line and rather work on the comment if it is irritating. Hope this explanation makes sense!

Reply:
Reverted. But, in my opinion, should be created an issue to clarify the comments.

Reply:
@3cpt do you have a suggestion for how the comment would be easier to understand?

You know best what you are looking for here, so by all means, feel free to raise a new ticket to clarify the comment.

Reply:
By removing the 0…1. Because the multiplier will be higher. But, yes, I believe that could be discussed in a future issue